app/src/main/java/clarkson/ee408/tictactoev4/MainActivity.java

```diff
@@ -94,6 +94,39 @@ public void requestMove() {
         });
     }
 
+    /**
+     * Sends the player's move to the server.
+     * @param move The move position (0-8) to send.
+     */
+    public void sendMove(int move) {
+        // Create a Request object with type SEND_MOVE
+        Request request = new Request();
+        request.setType(RequestType.SEND_MOVE);
+
+        // Serialize the move parameter using Gson
+        String serializedMove = gson.toJson(move);
+        request.setData(serializedMove);
```

Comment:
Can be simplified so that Gson isn't used
```suggestion
        request.setType(RequestType.SEND_MOVE);
        request.setData("" + move);
```